cst438_register/src/main/resources/application.properties

```diff
@@ -9,15 +9,16 @@ registration.url = http://localhost:8080
 registration.service = default
 
 # database properties
-# spring.datasource.url = jdbc:mysql://localhost:3306/register
-# spring.datasource.username = root
-# spring.datasource.password = dw4$Database
+ spring.datasource.url = jdbc:mysql://localhost:3306/register
+ spring.datasource.username = root
+ spring.datasource.password = root
```

Comment:
You accidentally pushed properties. It shows the password. Be careful not to commit properties if it isn't necessary.

Reply:
It is local so no harm no foul.

Reply:
Good catch, thanks I'll fix that up.